distribution/docs/src/main/resources/_contents/_quickstart/quickstart-configuring-contents.adoc

```diff
@@ -0,0 +1,15 @@
+
+=== Configuring (Quick Install)
+
+Set the configurations needed to run ${branding}
+
+.. Go to ${secure_url}/admin.
+.. Enter the default username of `admin` and the password of `admin`.
+.. Follow the prompts installer prompts for a standard installation.
+... Click start to begin the setup process.
+... Configure guest claims attributes or use defaults.
```

Comment:
I agree with you @stustison that we need to mention this. I'm just not sure this is the right place. Since this is a quick start guide, I would think that we want to keep the steps to a minimum and get the user started using only the defaults. We can always refer to section 2 here if needed. Which actually bring up the following question: shouldn't we provide more details about the installation process in section 2?